deployment/frontend/src/components/_shared/ActivityStreamCard.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Image from 'next/image'
 
-interface activity {
+export interface activity {
   description: string,
   time: string,
   icon: string
```

Comment:
I think these icons need to have a rounded background with a shadow 
![Uploading image.png…]()